src/main/java/seedu/address/logic/commands/TodayCommand.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.time.DayOfWeek;
+import java.time.LocalDate;
+
+import seedu.address.model.Model;
+
+public class TodayCommand extends Command {
```

Comment:
So this is a different feature from the one listed in this PR?

Reply:
It's also missing a javadoc description

Reply:
`c-tag-find` was implemented on top of  `c-today` feature, so `c-tag-find` branch contains the `c-today` feature.

Reply:
I've added the javadoc. Thanks!